assemblies/pentaho-solutions/src/main/resources/pentaho-solutions/system/applicationContext-spring-security.xml

```diff
@@ -406,6 +403,7 @@
         <sec:intercept-url pattern="\A/plugin/reporting/api/jobs/.*\Z" access="Anonymous,Authenticated" method="HEAD" />
         <sec:intercept-url pattern="\A/api/repos/.*\Z" access="Anonymous,Authenticated" method="OPTIONS" />
         <sec:intercept-url pattern="\A/api/repos/.*\Z" access="Anonymous,Authenticated" method="HEAD" />
+        <sec:intercept-url request-matcher-ref="requireJsConfigRequestMatcher" access="Anonymous,Authenticated" />
```

Comment:
📔  This security interceptor is mapped to `/api`, and this matcher handles Require JS config files being served by `/api/repos/...`.